### DISCUSSION TITLE
Fresh BirdNET installation detecting birds incorrectly (e.g. from a remote geography)

### DISCUSSION THREAD

**Discussion by jgelsey** [mcguirepr89/BirdNET-Pi#956] [Q&A] [answered]
Hi,

I've freshly installed BirdNET yesterday on a freshly-installed Raspbian Bullseye RPI4 system.  The system is detecting birds incorrectly - e.g. this morning it found a Blue-headed Parrot, even though I am in western Washington state.  I've run species.py and then rebooted to see if that might nudge the system to look at local species, but no luck.  

If it's useful at all for debugging this, the most common species detected is a Yellow-bellied Elaenia, which does not range further north than Mexico.

Thanks for any suggestions on what I might try here!

**Comment by ehpersonal38** [comment]
Could you try running this command and seeing if things come back to normal?

```
language=$(grep 'DATABASE_LANG=' ~/BirdNET-Pi/birdnet.conf | awk -F '=' '{print $2}') && chmod +x ~/BirdNET-Pi/scripts/install_language_label_nm.sh && ~/BirdNET-Pi/scripts/install_language_label_nm.sh -l $language
```

**Reply by tsiawd** [reply]
Mine is doing this also only finding birds from a different continent. Does anyone know a fix to this?

**Comment by jgelsey** [comment]
Thanks - unfortunately this doesn't seem to help.  

When I installed, I created a local version of newinstaller.sh (from `curl -s https://raw.githubusercontent.com/mcguirepr89/BirdNET-Pi/main/newinstaller.sh`) and put the `chmod +x` in there in order to get the original install to work.  It finished with no errors.

**Reply by ehpersonal38** [reply]
Can you uninstall then reinstall BirdNET-Pi? (scripts/uninstall.sh).

**Comment by jgelsey** [comment]
Done.  Doesn't seem to have made a difference, other than it's now detecting a "White-browed piculet" (native to Asia) with 75%-95% confidence.  Given that I am in Washington state, doesn't seem likely this is accurate. :)

Happy to give you ssh access to the RPI4 through a Tailscale VPN if you want to poke around - it's a fresh install of Bullseye so nothing sensitive there.

**Reply by ehpersonal38** [reply]
Sure, that sounds good. Send me an email with the details and I'll have a look. My email is my Github username @gmail.com.

**Reply by ehpersonal38** [reply, marked as answer]
Okay, got your system all fixed up! Turns out you were using the old model so the labels file was mismatched. I just changed the model to the new one and all is well. You also had some weird issue after that where the analysis was hanging... I still don't know why it happened, but a few restarts of services and it seems to be chugging away again. Let me know if you need any other assistance!

**Reply by rovermeyer** [reply]
What is the fix for this?  I did a new install yesterday. I live in Maryland and all of the birds I'm detecting are from Africa.

**Reply by Quake9203** [reply]
Same issue. I just installed earlier this week. I live in Michigan and am detecting birds in China and Vietnam.

**Comment by jgelsey** [comment]
Awesome - thank you!  Super appreciate you fixing it up for me.  We just
moved to a very rural area with all sorts of interesting birds to track, so
BirdNET will be super useful to us.

**Comment by vinceskahan** [comment]
(apologies - solved this one myself by switching to the BirdNET_GLOBAL_6K_V2.4_Model_FP16 (2023) database via the Tools menu.  Should that be the default these days ???)

~~ Me too.  Also near Seattle/Tacoma.  Merlin on the iPhone is correctly identifying the white-crowned sparrow that I'm sitting here looking at, but BirdNET on the pi.is incorrectly identifying it as a Malagasy Bulbul or a Long-tailed Koel, neither or which are from anywhere near here.  This is on a clean pi4 using latest raspi os 64 bit. ~~

**Reply by SharonD214** [reply]
Thanks, this solved my issue as well

**Comment by kpcmm** [comment]
I've had the same problem, and hopefully solved it by switching to the new model. Is there any way to get a list of corresponding names for the new model. E.g. what should a blue headed parrot really be? I'd like to have a go at updating the detections table with the correct names and renaming all those mp3 files to match. I have over a week of data which it would be a shame to lose. Did this data get uploaded somewhere? Is bad data percolating up? Thanks, Kevin

**Comment by vinceskahan** [comment]
Read the thread. Switch which db you use.

**Reply by rovermeyer** [reply]
I don't see any instructions on how to change DB's

**Reply by rovermeyer** [reply]
Found it under settings.

Thanks

**Reply by illflavored** [reply]
This happened to me as well
The fix is to go to Tools --> Settings
Under this you will see **Model** at the top with a dropdown to `Select a Model:`

It defaults to `BirdNET_6K_GLOBAL_MODEL` which is legacy. You don't want this one. It's described as

> BirdNET_6K_GLOBAL_MODEL (2020)
> This is the BirdNET-Lite model, with bird sound recognition for more than 6,000 species worldwide. This has generally worse performance than the newer models but is kept as a legacy option.

The fix is to switch the model to

> BirdNET_GLOBAL_6K_V2.4_Model_FP16 (2023)
> This is the BirdNET-Analyzer model, the most advanced BirdNET model to date. Currently it supports over 6,000 species worldwide, giving quite good species coverage for people in most of the world.

Then don't forget to hit Update Settings at the bottom of the page to save/restart services

I think it would make sense to change the default to the current instead of legacy on new installs

**Comment by srd424** [comment]
@ehpersonal38 :
> You also had some weird issue after that where the analysis was hanging... I still don't know why it happened, but a few restarts of services and it seems to be chugging away again. Let me know if you need any other assistance!

I've seen an issue fairly frequently where incomplete recordings cause analysis to hang, but I've never managed to capture a good example for debugging!

**Comment by Candyjet** [comment]
I had the same problem. Switching the model fixed it, thanks.

**Comment by AlaskanAstro** [comment]
I had the same issue. Seems like the default model should be changed if people keep having this issue?

**Reply by macjustice** [reply]
Just ran into it myself, reviving my birdnetpi after a year or so off. Glad the fix is easy, but definitely change the default
